src/patternfly/components/Select/examples/select-single-top-expanded-example.hbs

```diff
@@ -0,0 +1,3 @@
+{{#> select id="select-single-top-expanded" select--modifier="pf-m-top" select--IsExpanded="true"}}
```

Comment:
using `select--modifier="pf-m-top"` is adding `.pf-m-top` to both `.pf-c-select` (not needed) and to `.pf-c-select__menu` (the element that needs it). Can you update `select-multi.hbs` and `select-single.hb` to use `{{#if select-menu--modifier}} {{select-menu--modifier}}{{/if}}` instead of `{{#if select--modifier}} {{select--modifier}}{{/if}}`, and then change `select--modifier="pf-m-top"` to `select-menu--modifier="pf-m-top"` in this file?

FWIW I completely forgot about this PR (doh!!!) and opened another with the changes if you want to reference - https://github.com/patternfly/patternfly-next/pull/2164/files

There is also a typo in `select-multi.hbs` referenced in #2164 if you want to fix that, too.